examples/changestream.php

```diff
@@ -53,7 +50,7 @@ function toJSON(object $document): string
     $changeStream->next();
 
     if (time() - $startTime > 3) {
-        fprintf(STDERR, "Aborting after 3 seconds...\n");
+        printf("Aborting after 3 seconds...\n");
```

Comment:
Necessary to avoid PHPUnit considering the tests failed due to output, and to allow asserting against the full test output.